### PR TITLE
Bug 1797701: allow changing the storage class back to the default one

### PR DESCRIPTION
**Pull request by suomiy**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/selectors/config-map/sc-defaults.ts [hunk 1]
```diff
@@ -19,24 +19,29 @@ const getSCConfigMapAttribute = (
...
+    'accessMode',
+    PVC_ACCESSMODE_DEFAULT,
   );
+
+export const getDefaultSCAccessModes = (
+  storageClassConfigMap: ConfigMapKind,
+  storageClassName: string,
+) => {
+  const defaultMode = getDefaultSCAccessMode(storageClassConfigMap, storageClassName);
+  return defaultMode ? [defaultMode] : [];
```
Comment (mareklibra) [inline frontend/packages/kubevirt-plugin/src/selectors/config-map/sc-defaults.ts]: Considering implementation of the `getDefaultSCAccessMode`, this condition is not needed.
I would even merge these two `getDefaultSCAccessMode() and getDefaultSCAccessModes()` into a single function.

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/selectors/config-map/sc-defaults.ts]: you are right, fixed

File: frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/initialize-vm.ts [hunk 2]
```diff
@@ -14,88 +14,83 @@ import { NetworkWrapper } from '../../../wrapper/vm/network-wrapper';
...
 import { VM_TEMPLATE_NAME_PARAMETER } from '../../../../constants/vm-templates';
 import { CreateVMEnhancedParams } from './types';
 
+const resolveDataVolumeName = (
+  dataVolumeWrapper: DataVolumeWrapper,
+  {
+    newName,
+    isTemplate,
+    isPlainDataVolume,
+  }: { newName: string; isTemplate: boolean; isPlainDataVolume: boolean },
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/initialize-vm.ts]: Just wondering, why do we need to wrap these arguments in an object?

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/initialize-vm.ts]: IMO, it is easier to navigate the code like this (eg https://github.com/openshift/console/pull/4217/files#diff-1a53f6615b700761ee940794dc53a83eR72)
first argument is base for the name and the second are the options which affect the name resolution

**Pull request merged**